fieldservice_sale/models/sale_order_line.py

```diff
@@ -8,6 +8,8 @@
 class SaleOrderLine(models.Model):
     _inherit = "sale.order.line"
 
+    qty_delivered_method = fields.Selection(
+        selection_add=[('field', 'Field Service Order')])
```

Comment:
'field' is unclear. may be use field_service ?

Reply:
Sounds good to me